### ISSUE TITLE
Error in console when i use broomstick( Default Configuration | Minecraft 1.9.4 )

### ISSUE THREAD

**Issue by VolanDeVovan** [elBukkit/MagicPlugin#139]
Spigot version: CraftBukkit version git-Spigot-798f32d-0cd0397 (MC: 1.9.4) (Implementing API version 1.9.4-R0.1-SNAPSHOT)
Plugin version: 6.2(latest)

```
[16:31:03] [Server thread/WARN]: java.lang.IllegalArgumentException: Can not set boolean field net.minecraft.server.v1_9_R2.EntityArmorStand.bz to java.lang.Integer
[16:31:03] [Server thread/WARN]: 	at sun.reflect.UnsafeFieldAccessorImpl.throwSetIllegalArgumentException(UnsafeFieldAccessorImpl.java:167)
[16:31:03] [Server thread/WARN]: 	at sun.reflect.UnsafeFieldAccessorImpl.throwSetIllegalArgumentException(UnsafeFieldAccessorImpl.java:171)
[16:31:03] [Server thread/WARN]: 	at sun.reflect.UnsafeBooleanFieldAccessorImpl.set(UnsafeBooleanFieldAccessorImpl.java:86)
[16:31:03] [Server thread/WARN]: 	at java.lang.reflect.Field.set(Field.java:764)
[16:31:03] [Server thread/WARN]: 	at com.elmakers.mine.bukkit.utility.CompatibilityUtils.setDisabledSlots(CompatibilityUtils.java:925)
[16:31:03] [Server thread/WARN]: 	at com.elmakers.mine.bukkit.spell.builtin.LevitateSpell.configureArmorStand(LevitateSpell.java:972)
[16:31:03] [Server thread/WARN]: 	at com.elmakers.mine.bukkit.spell.builtin.LevitateSpell.onActivate(LevitateSpell.java:888)
[16:31:03] [Server thread/WARN]: 	at com.elmakers.mine.bukkit.spell.BaseSpell.setActive(BaseSpell.java:2016)
[16:31:03] [Server thread/WARN]: 	at com.elmakers.mine.bukkit.magic.Mage.activateSpell(Mage.java:1259)
[16:31:03] [Server thread/WARN]: 	at com.elmakers.mine.bukkit.spell.BaseSpell.activate(BaseSpell.java:2027)
[16:31:03] [Server thread/WARN]: 	at com.elmakers.mine.bukkit.spell.builtin.LevitateSpell.onCast(LevitateSpell.java:629)
[16:31:03] [Server thread/WARN]: 	at com.elmakers.mine.bukkit.spell.BaseSpell.finalizeCast(BaseSpell.java:1238)
[16:31:03] [Server thread/WARN]: 	at com.elmakers.mine.bukkit.spell.BaseSpell.cast(BaseSpell.java:1121)
[16:31:03] [Server thread/WARN]: 	at com.elmakers.mine.bukkit.spell.BaseSpell.cast(BaseSpell.java:942)
[16:31:03] [Server thread/WARN]: 	at com.elmakers.mine.bukkit.spell.BaseSpell.cast(BaseSpell.java:1706)
[16:31:03] [Server thread/WARN]: 	at com.elmakers.mine.bukkit.wand.Wand.cast(Wand.java:3686)
[16:31:03] [Server thread/WARN]: 	at com.elmakers.mine.bukkit.wand.Wand.cast(Wand.java:3668)
[16:31:03] [Server thread/WARN]: 	at com.elmakers.mine.bukkit.wand.Wand.performAction(Wand.java:4908)
[16:31:03] [Server thread/WARN]: 	at com.elmakers.mine.bukkit.magic.listener.PlayerController.onPlayerInteract(PlayerController.java:431)
[16:31:03] [Server thread/WARN]: 	at sun.reflect.GeneratedMethodAccessor30.invoke(Unknown Source)
[16:31:03] [Server thread/WARN]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[16:31:03] [Server thread/WARN]: 	at java.lang.reflect.Method.invoke(Method.java:498)
[16:31:03] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306)
[16:31:03] [Server thread/WARN]: 	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62)
[16:31:03] [Server thread/WARN]: 	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:502)
[16:31:03] [Server thread/WARN]: 	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:487)
[16:31:03] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_9_R2.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:231)
[16:31:03] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_9_R2.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:198)
[16:31:03] [Server thread/WARN]: 	at net.minecraft.server.v1_9_R2.PlayerInteractManager.a(PlayerInteractManager.java:111)
[16:31:03] [Server thread/WARN]: 	at net.minecraft.server.v1_9_R2.PlayerConnection.a(PlayerConnection.java:831)
[16:31:03] [Server thread/WARN]: 	at net.minecraft.server.v1_9_R2.PacketPlayInBlockDig.a(SourceFile:40)
[16:31:03] [Server thread/WARN]: 	at net.minecraft.server.v1_9_R2.PacketPlayInBlockDig.a(SourceFile:10)
[16:31:03] [Server thread/WARN]: 	at net.minecraft.server.v1_9_R2.PlayerConnectionUtils$1.run(SourceFile:13)
[16:31:03] [Server thread/WARN]: 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[16:31:03] [Server thread/WARN]: 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[16:31:03] [Server thread/WARN]: 	at net.minecraft.server.v1_9_R2.SystemUtils.a(SourceFile:45)
[16:31:03] [Server thread/WARN]: 	at net.minecraft.server.v1_9_R2.MinecraftServer.D(MinecraftServer.java:726)
[16:31:03] [Server thread/WARN]: 	at net.minecraft.server.v1_9_R2.DedicatedServer.D(DedicatedServer.java:399)
[16:31:03] [Server thread/WARN]: 	at net.minecraft.server.v1_9_R2.MinecraftServer.C(MinecraftServer.java:665)
[16:31:03] [Server thread/WARN]: 	at net.minecraft.server.v1_9_R2.MinecraftServer.run(MinecraftServer.java:564)
[16:31:03] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:745)
```

**Comment by NathanWolf** [comment]
Looks like a compatibility issue I may need to resolve- I think if you try Magic 6.0.4 it should work with your version of Spigot though.

**Comment by VolanDeVovan** [comment]
If I am not mistaken in the old version available dupe items.
Error Is corrected in newer versions?
(sorry for my English)

**Comment by NathanWolf** [comment]
I'm not aware of any dupe bugs. If there was one, it's probably not fixed- but no one has reported anything like that.

**Comment by VolanDeVovan** [comment]
this bug will be fixed in the new version?

**Comment by NathanWolf** [comment]
The broomstick bug I need to fix. Whatever dupe bug you're taking about I'm not aware of so can't fix unless I get some details on it.

**Comment by VolanDeVovan** [comment]
In the new version of the plugin does not support dupe because now call the list of skills occurs at q button!

**Comment by NathanWolf** [comment]
In that case any version above 6 should have that same fix.

Sounds like that might have been due to a compatibility issue with another plugin, though, that's not normal behavior.

**Comment by VolanDeVovan** [comment]
This problem may occur on a server with 1 plugin!

**Comment by MrTrojan** [comment]
Hello. my gays :+1:

**Comment by NathanWolf** [comment]
Brooms should be fixed in 6.1.2. Tested in 1.10 only, please re-open this issue if they are still broken in 1.9.
